lib/movements.js

```diff
@@ -92,6 +135,15 @@ class Movements {
     for (const shape of b.shapes) {
       b.height = Math.max(b.height, pos.y + dy + shape[4])
     }
+
+    // check if open door, if so treat it like air so the bot can navigate
+    if (b.name.includes('door') && b.material === 'wood' && doorOpenMetadata.includes(b.metadata)) {
```

Comment:
Also worth noting that since the open door still has a collision box, there is a chance the bot may run into it from the corner in certain cases by treating it like air. This would need to be tested.